### PR TITLE
Disclaimer decline button text is not visible in safari

### PR DESCRIPTION
**Pull request by giladt**
Apparently the `-webkit-text-fill-color: transparent;` is what’s failing in Safari, but only when used with a `button` tag. Once I change the button into a div tag (and set it to `inline-block`) it functions and looks correctly on chrome/safari/opera/edge/brave/firefox browsers.

I found this [issue](https://www.notion.so/curvelabs/Disclaimer-DECLINE-button-text-is-not-visible-in-Safari-16ef63b72fa0480296dedd79744cd998) affecting both the **disclaimer** and the **stage4** page of the wizard.

### CODE REVIEW

File: src/home/home.scss [hunk 1]
```diff
@@ -323,18 +323,3 @@
     }
   }
 }
-
-@media screen and (max-width: 420px) {
```
Comment (dkent600) [inline src/home/home.scss:327]: This should be there, no?  Was part of a previous PR?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/home/home.scss (hunk 1 above)
Comment (giladt) [reply src/home/home.scss:327]: No, this part is from the previous PR. I'm not sure why it has been recommitted, but it should not be included in the code.

Comment (dkent600) [reply src/home/home.scss:327]: Can you remove it then?

**Review by dkent600 — CHANGES_REQUESTED** [review]

**Comment by giladt** [comment]
Created a new clean PR #126

**Pull request closed without merging**